### DISCUSSION TITLE
ERROR: No matching distribution found for setuptools

### DISCUSSION THREAD

**Discussion by ozmel1** [mindsdb/mindsdb#5235] [Get help] [answered]
Hi, 

I am following steps in https://github.com/mindsdb/mindsdb/discussions/2108 and getting an error at 

Install pip install -e .

 ERROR: Could not find a version that satisfies the requirement setuptools (from versions: none)
 ERROR: No matching distribution found for setuptools

I have upgraded both pip and setuptools. Still the same error

OS: Windows 10. Python 3.10.10

Much appreciate any help to resolve this!

**Comment by ZoranPandovski** [comment, marked as answer]
Hi @ozmel1, python 3.10 is not a supported version. Please use 3.7.x, 3.8.x, or 3.9.x
